docs/REST API/rest-api.md

```diff
@@ -8736,6 +8738,95 @@ Gets the result of a script that was executed.
 
 > Note: `created_at` is the creation timestamp of the script execution request.
 
+### Run bulk script
+
+Run a script on multiple hosts.
+
+The script will be added to each host's list of upcoming activities.
+
+`POST /api/v1/fleet/scripts/bulk`
+
+#### Parameters
+
+| Name            | Type    | In   | Description                                                                                    |
+| ----            | ------- | ---- | --------------------------------------------                                                   |
+| host_ids        | array   | body | **Required**. List of host IDs.                                                |
+| script_id       | integer | body | The ID of the existing saved script to run. |
+
+
+#### Example
+
+`POST /api/v1/fleet/scripts/bulk`
+
+##### Request body
+
+```json
+{
+  "script_id": 123,
+  "host_ids": [1, 2, 3]
+}
+```
+
+##### Default response
+
+`Status: 202`
+
+
+```json
+{
+  "batch_execution_id": "e797d6c6-3aae-11ee-be56-0242ac120002"
```

Comment:
Do we want to call this **bulk** or **batch**?  We should stick with one.

Reply:
`batch` 👍